src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -202,4 +212,16 @@ internal class RecommendationService(
       )
     )
   }
+
+  private fun fetchMappaCategory(riskResponse: RiskResponse?): String {
```

Comment:
These 2 methods feel like they are more suited to the `RecommendationToPartADataMapper` class as they seem to specifically be around around how to format the text for the Part A.